pyproject.toml

```diff
@@ -1,7 +1,36 @@
-[tool.isort]
-profile = "black"
-lines_after_imports = 2
-src_paths = ["kinto-remote-settings", "tests"]
+[tool.ruff]
+# Exclude a variety of commonly ignored directories.
+exclude = [
+    ".git",
+    "__pycache__",
+    ".venv",
+    "node_modules",
+]
+
+[tool.ruff.lint]
+preview = true
+select = [
+    # pycodestyle
+    "E", "W",
+    # flake8
+    "F",
+    # isort
+    "I"
+]
+ignore = []
+
+[tool.ruff.lint.isort]
+lines-after-imports = 2
+
+[tool.ruff.format]
+# Like Black, use double quotes for strings.
+quote-style = "double"
+# Like Black, indent with spaces, rather than tabs.
+indent-style = "space"
+# Like Black, respect magic trailing commas.
+skip-magic-trailing-comma = false
+# Like Black, automatically detect the appropriate line ending.
+line-ending = "auto"
```

Comment:
This looks the same as what comes in the default configuration. Can we get away with not specifying it?

Reply:
My bad